### PR TITLE
Making the users and hearts pretty

### PR DESCRIPTION
**Pull request by markschuttehva**

### CODE REVIEW

File: app/static/scripts/main.js [hunk 1]
```diff
@@ -1,39 +1,39 @@
...
-          .register("/sw.js")
-          .then(registration => {
-              console.log(`ServiceWorker registration successfull with scope: ${registration.scope}`);
-          })
-          .catch(() => {
-              console.log("Failed to register ServiceWorker");
-          });
-  });
-}
+// // Register serviceworker
```
Comment (kriskuiper) [inline app/static/scripts/main.js]: You forgot to comment this back on

File: app/static/styles/main.css [hunk 2]
```diff
@@ -327,6 +327,65 @@ nav {
     margin: 1em 0;
 }
 /* 404 Page styling */
+@media screen and (max-width: 120rem) {
```
Comment (kriskuiper) [inline app/static/styles/main.css:409]: Use min-width instead

File: app/static/styles/main.css [hunk 3]
```diff
@@ -327,6 +327,65 @@ nav {
...
+
+@media screen and (max-width: 90rem) {
+  .card-grid {
+  grid-template-columns: repeat(3, 1fr);
+  }
+}
+
+@media screen and (max-width: 60rem) {  
+  .card-grid {
+  grid-template-columns: repeat(2, 1fr);
```
Comment (kriskuiper) [inline app/static/styles/main.css:423]: Indentation not on point

Comment (kriskuiper) [reply app/static/styles/main.css:423]: On every grid

File: app/static/styles/main.css [hunk 4]
```diff
@@ -327,6 +327,65 @@ nav {
...
+    text-align: center;
+    font-size: 4em;
+    margin: 0em;
+}
+
+main {
+    margin:1em 1em 0em 1em;
+}
+
+main section a article form {
```
Comment (kriskuiper) [inline app/static/styles/main.css:443]: Use BEM principles instead of html selectors

File: app/static/styles/main.css [hunk 5]
```diff
@@ -327,6 +327,65 @@ nav {
...
+    z-index: 1;
+    position: relative;
+}
+
+main section a article section {
+    text-align: center;
+    margin-top: 1.5em;
+}
+
+main section a article form button {
```
Comment (kriskuiper) [inline app/static/styles/main.css:454]: BEM

File: app/static/styles/main.css [hunk 6]
```diff
@@ -327,6 +327,65 @@ nav {
...
+    margin:1em 1em 0em 1em;
+}
+
+main section a article form {
+    margin-top: -5em;
+    z-index: 1;
+    position: relative;
+}
+
+main section a article section {
```
Comment (kriskuiper) [inline app/static/styles/main.css:449]: BEM

File: app/static/sw.js [hunk 7]
```diff
@@ -1,28 +1,28 @@
...
-    "./index.html",
-    "./404.html",
-    "./401.html",
-    "./scripts/main.js",
-    "./styles/main.css",
-    "./media/images/icons/icon-192.png",
-    "./media/images/icons/icon-512.png",
-    "./manifest.webmanifest"
-];
+// const CACHE_NAME = "drinkerz-app-cache";
```
Comment (kriskuiper) [inline app/static/sw.js]: Forgot to comment this out

File: app/static/sw.js [hunk 8]
```diff
@@ -1,28 +1,28 @@
...
 
-// Install service worker
-self.addEventListener("install", event => {
-    console.log("[ServiceWorker] Caching app shell"); // eslint-disable-line
-    event.waitUntil(
-        caches.open(CACHE_NAME)
-            .then(cache => cache.addAll(urlsToCache))
-    );
-});
+// // Install service worker
```
Comment (kriskuiper) [inline app/static/sw.js]: Again, comment this on

File: app/view/users.ejs [hunk 9]
```diff
@@ -1,4 +1,5 @@
 <% include partials/head.ejs %>
+<meta content="width=device-width, initial-scale=1" name="viewport" />
```
Comment (kriskuiper) [inline app/view/users.ejs:2]: This already is in `partials/head.ejs`

**Review by kriskuiper — CHANGES_REQUESTED** [review]

**Review by Aronsmit2 — APPROVED** [review]

**Review by kriskuiper — APPROVED** [review]

**Pull request merged**